flow/util/genMetrics.py

```diff
@@ -814,6 +820,18 @@ def extract_metrics(cwd, platform, design, flow_variant, output, hier_json):
                                         'hold violation count (\S+)'),
                        logPath + '/6_report.log')
 
+    extractTagFromFile('finish__timing__cp_delay',
+                       metrics_dict,
+                       baseRegEx.format('finish critical path delay',
+                                        '(\S+)'),
+                       logPath + '/6_report.log')
+
+    extractTagFromFile('finish__timing__cp_slack_per_delay',
```

Comment:
The name is a bit cryptic - is this critical path slack as a percent of path delay?  Maybe 

critical_path_slack_as_percent_path_delay

@ravi-varadarajan any comment from metrics 2.1 naming?

Reply:
The abbreviation was suggested by Ravi on a chat on Slack. His argument was that the descriptor was getting too long.

Reply:
maybe finish__timing__wns_percent_delay ?  @ravi-varadarajan

Reply:
If this metrics is trying to capture the critical path slack as a pct of the total delay of the path what @maliberty suggested will work  "finish__timing__wns_percent_delay"  should be fine

Reply:
@maliberty, Ravi confirmed on Slack that the rename is ok. Fixed on [06a180d](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/456/commits/06a180da5d6517817e73f0a39e8fec8401ac0432)